src/files.js

```diff
@@ -486,5 +486,8 @@ FOAM_FILES([
   { name: "foam/crypto/hash/Hashable" },
   { name: "foam/crypto/sign/Signer" },
   { name: "foam/crypto/sign/Signable" },
-  { name: "foam/crypto/sign/SignedFObject" }
+  { name: "foam/crypto/sign/SignedFObject" },
+
+  //Help Desk Support Files
+  { name: "foam/support/model/MessageModel"}
```

Comment:
We shouldn't add support models to the files.js as that will make all FOAM apps larger, even those that don't use the support features. Instead, create a new support.js file list to contain all of the support files. Like what we do for nanos.js.